.docker/scripts/create-test-data.sh

```diff
@@ -76,6 +70,12 @@ curl -LsH "Authorization: ${API_KEY}" \
 # END.
 #
 
+# Creating test data hierarchy which creates organisations assigned to datasets
+ckan_cli create-test-data hierarchy
```

Comment:
is this to match upstream?

Reply:
Not upstream, but other repos belonging to us.